js/package.json

```diff
@@ -1,18 +1,18 @@
 {
-  "name": "@fof/user-bio",
-  "version": "0.0.0",
-  "dependencies": {
-    "flarum-webpack-config": "^0.1.0-beta.8",
-    "autolink-js": "1.0.2",
-    "webpack": "^4.0.0",
-    "webpack-cli": "^3.0.7"
-  },
-  "scripts": {
-    "build": "webpack --mode production",
-    "watch": "webpack --mode development --watch",
-    "lint": "prettier --single-quote --trailing-comma es5 --print-width 150 --tab-width 4 --write 'src/**/*'"
-  },
-  "devDependencies": {
-    "prettier": "^1.15.2"
-  }
+    "private": true,
+    "name": "@fof/user-bio",
+    "dependencies": {
+        "flarum-webpack-config": "^0.1.0-beta.8",
+        "autolink-js": "1.0.2",
+        "webpack": "^4.0.0",
+        "webpack-cli": "^3.0.7"
+    },
+    "scripts": {
+        "dev": "webpack --mode development --watch",
+        "build": "webpack --mode production",
+        "lint": "prettier --single-quote --trailing-comma es5 --print-width 150 --tab-width 4 --write 'src/**/*'"
+    },
+    "devDependencies": {
+        "prettier": "^1.15.2"
```

Comment:
Would prettier v2 be a more appropriate choice?

Reply:
Not sure. I've kept the original configuration that was already in the package (but it hadn't be used it seems so...)

Reply:
I guess it does not really matter either way :)

Reply:
It's not a focus of this PR, so it's fine not updating. It'd also mean all files with arrow functions & one arg would have changes, so let's avoid that :)